app/templates/users/edit.html

```diff
@@ -1,13 +1,57 @@
 {% extends "layouts/one-column.html" %}
+{% from "includes/forms.html" import checkbox_field, delete_button, fieldset %}
 
 {% set page_title = "User: " + user.username %}
+{% set breadcrumbs = [
+  {text: 'Home', url: '/'},
+  {text: 'Users', url: '/#Users'},
+  {text: 'User: ' + user.username}
+] %}
+{% set debug_items = [
+  {name: 'user', value: user},
+  {name: 'apps_options', value: apps_options},
+  {name: 'buckets_options', value: buckets_options}
+] %}
 
 {% block main_column %}
 
-<pre>user = {{ user | dump(4) | safe }}</pre>
-<br />
-<pre>apps_options = {{ apps_options | dump(4) | safe }}</pre>
-<br />
-<pre>buckets_options = {{ buckets_options | dump(4) | safe }}</pre>
+<form action="{{ url_for('users.edit', { id: user.auth0_id | urlencode }) }}" method="post">
 
+  {% call fieldset(
+    "superadmin",
+    "Super Admin",
+    hint="User is a super admin, allowing all privileges on the Control panel."
+  ) -%}
+    <div class="govuk-checkboxes">
+      {{ checkbox_field("superadmin", "Super Admin", user.is_superuser) }}
+    </div>
+  {%- endcall %}
+
+  <div class="form-group">
+    <input type="submit" class="button button-secondary" value="Save changes">
+  </div>
+
+</form>
+
+<form method="{{ url_for('users.reset_mfa', { id: user.auth0_id | urlencode }) }}" method="post">
+
+  {% call fieldset(
+    "resetmfa",
+    "Reset MFA",
+    hint="Reset the user's multi-factor authentication settings, forcing them to reconfigure."
+  ) -%}
+    <div class="form-group">
+      <input type="submit" class="button button-secondary" value="Reset MFA">
+    </div>
+  {%- endcall %}
+
+</form>
+
+<hr>
+
+<form action="{{ url_for("users.delete", { id: user.auth0_id }) }}" method="post">
```

Comment:
It would good to explain that this is useful as a tidy up only if the user is removed from the github org, otherwise they can simply log-in again and their user is recreated. Maybe we should also say whether this delete also deletes their app bucket permissions, AWS role and NFS home directory. This could all be a follow up ticket.

Reply:
Deleting users is outside the scope of this change - deleting bucket permissions, etc is a change for the control panel api, not the frontend. I just put the delete user page on the user details page as it seems like a sensible place for it - it already exists on the user list page.